tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.Service/Microsoft.Legal.MatterCenter.Repository/SharePointInfo/Taxonomy.cs

```diff
@@ -189,7 +189,7 @@ private TaxonomyResponseVM GetTaxonomyHierarchy (TermStore termStore, TermStoreD
                         //This condition is if the UI is requesting practice group terms
                         if (termStoreDetails.TermSetName == taxonomySettings.PracticeGroupTermSetName)
                         {
-                            taxonomyResponseVM.TermSets = GetPracticeGroupTermSetHierarchy(clientContext, termSet, termStoreDetails);
+                            taxonomyResponseVM.TermSets = GetManagedTermSetHierarchy(clientContext, termSet, termStoreDetails);
```

Comment:
Changed the method name from GetPracticeGroupTermSetHierarchy to GetManagedTermSetHierarchy